src/main/java/exceptions/TableNotExist.java

```diff
@@ -0,0 +1,7 @@
+package exceptions;
+
+public class TableNotExist extends Exception{
```

Comment:
Space before {

Reply:
Same comment in every Exception class

Reply:
Also extends from RuntimeException.

also what is the difference between RuntimeException and Exception ?

Reply:
ok will modify those

Reply:
ok new information,
i searched for the difference.
what I get is that 
runtimeExceptoin is an unchecked exception that doesn't make a compilation error when I write it in my code unlike excepting it is checked one and make compilation error and I think runtime is part of exception